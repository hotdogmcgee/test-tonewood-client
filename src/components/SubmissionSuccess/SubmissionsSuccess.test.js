import React from "react";
import SubmissionsSuccess from "./SubmissionSuccess";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

describe("SubmissionsSuccess component", () => {
  it("renders a SubmissionsSuccess by default", () => {
    const wrapper = shallow(<SubmissionsSuccess />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});