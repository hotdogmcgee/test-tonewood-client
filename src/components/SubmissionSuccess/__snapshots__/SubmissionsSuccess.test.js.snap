// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubmissionsSuccess component renders a SubmissionsSuccess by default 1`] = `
<Fragment>
  <h1>
    Submission successful!
  </h1>
  <Link
    className="go-back-link link"
    to="/"
  >
    Go back to main page
  </Link>
  <Link
    className="new-submission-link link"
    onClick={[Function]}
    to="/new-submission"
  >
    Enter a new submission
  </Link>
</Fragment>
`;
